Dockerfile

```diff
@@ -14,14 +14,8 @@ RUN apt-get update && apt-get install -y \
 # import previously downloaded packages
 WORKDIR /root/control_lib
 # install libraries and dependencies
-COPY ./source/state_representation ./state_representation
-RUN /bin/bash -c "source ./state_representation/install.sh"
-
-COPY ./source/dynamical_systems ./dynamical_systems
-RUN /bin/bash -c "source ./dynamical_systems/install.sh"
-
-COPY ./source/robot_model ./robot_model
-RUN /bin/bash -c "source ./robot_model/install.sh"
+COPY ./source ./
+RUN /bin/bash -c "source ./install.sh"
```

Comment:
This step seems to work fine. Everything builds, apart from a few specific tests in robot_model. 

But, in line 55,
`COPY --from=builder --chown=${USER} /root/control_lib ./control_lib`
When I spin up and enter the container with `sh run.sh`, I don't see any `control_lib/lib/googletest` even though I know for a fact that it's cloned into the `/root/control_lib/lib...` directory which is copied. Any thoughts why this wouldn't be copied properly? @buschbapti